tests/python/unittest/test_image.py

```diff
@@ -139,6 +140,26 @@ def test_resize_short(self):
                     mx_resized = mx.image.resize_short(mx_img, new_size, interp)
                     assert_almost_equal(mx_resized.asnumpy()[:, :, (2, 1, 0)], cv_resized, atol=3)
 
+    @with_seed()
+    def test_imresize(self):
+        try:
+            import cv2
+        except ImportError:
+            raise unittest.SkipTest("Unable to import cv2")
+        for img in TestImage.IMAGES:
+            cv_img = cv2.imread(img)
+            mx_img = mx.nd.array(cv_img[:, :, (2, 1, 0)])
+            for _ in range(3):
```

Comment:
why test the same things three times here?

Reply:
i based it on the `test_resize_short` method in line 131. I can remove it, should be okay either ways.